seatunnel-config/src/test/java/org/apache/seatunnel/config/CompleteTests.java

```diff
@@ -15,12 +15,7 @@
  * limitations under the License.
  */
 
-package interestinglab;
-
-import org.apache.seatunnel.config.Config;
-import org.apache.seatunnel.config.ConfigFactory;
-import org.apache.seatunnel.config.ConfigRenderOptions;
-import org.apache.seatunnel.config.ConfigResolveOptions;
```

Comment:
These lines don’t seem to need to be deleted, because the following code is in use.

Reply:
> These lines don’t seem to need to be deleted, because the following code is in use.

because the following variables are never used. 😅

Reply:
This is not a necessary import.

Reply:
So, should the useless content in the main method be deleted?

Reply:
in fact, just from myside, we can remove all files under this package. cc @CalvinKirs @wuchunfu @garyelephant 

- manys files are marked unused.
- the unit is coming https://github.com/InterestingLab/seatunnel/issues/783

![image](https://user-images.githubusercontent.com/95013770/147333956-95ac6a6c-2a0a-40d3-a9c4-ccd4c6f97ce3.png)

Reply:
Yes, I am doing this work, and currently we have directly copied the code related to typesafe.